basket/news/views.py

```diff
@@ -31,9 +30,7 @@
 from basket.news.models import (
     CommonVoiceUpdate,
     Interest,
-    LocaleStewards,
```

Comment:
Are these two unimported models still in use elsewhere? If not, would be good too throw them in the woodchipper while we're at it?